### PR TITLE
Use openssh export keys for x2go on windows

### LINKED ISSUE #335: X2Go private key issue on Windows
Linux (RHEL7 and Co.) generated private keys are not understood by X2Go (it prompts for a passphrase indpepndetly whether the key is passphrase protected or not). 

The soution is either to use pageant (using windows style .ppk format), or convert the private key to "older" format (like it was in RHEL6 and Co.) using puttygen. (conversions -> export SSH key), but do not use the force new format.

X2Go does not understand windows style (.ppk) key format.

The background:
The new format uses a header (someting like openssh-key-v1nonenonessh-rsa in the base64 encoded key, the old format strictly have the key). 

I have no idea wether the MacOS or the Linux versions are also work the same way.

### PR DESCRIPTION
**Pull request by stdweird**
Fixes #335

### CODE REVIEW

File: intro-HPC/ch_account.tex [hunk 1]
```diff
@@ -133,13 +133,25 @@ \subsection{How do SSH keys work?}
...
 
-    \item  Finally, save both the public and private keys in a folder on your
+    \item  Save both the public and private keys in a folder on your
       personal computer (We recommend to create and put them in the folder
       ``C:\textbackslash Users\textbackslash \%USERNAME\%\textbackslash
       AppData\textbackslash Local\textbackslash PuTTY\textbackslash .ssh'')
       with the buttons \keys{Save public key} and \keys{Save private key}.
       We recommend using the name ``\strong{id\_rsa.pub}'' for the public key,
       and ``\strong{id\_rsa.ppk}'' for the private key.
+    \item Finally save an ``OpenSSH'' version of your private key (in particular for later ``X2Go'' usage)
```
Comment (itkovian) [inline intro-HPC/ch_account.tex]: `Finally,`

File: intro-HPC/ch_account.tex [hunk 2]
```diff
@@ -133,13 +133,25 @@ \subsection{How do SSH keys work?}
...
+      by entering the ``Conversions'' menu and selecting ``Export OpenSSH key''
+      (do \strong{not} select the ``force new file format'' variant).
+      Save the file in the same location as in the previous step
+      with filename ``\strong{id_rsa}''.
+      (If there is not ``Conversions'' menu, you need to update your ``puttygen'' version.
```
Comment (itkovian) [inline intro-HPC/ch_account.tex]: - s/not/no/
- s/need to/should/

Comment (stdweird) [reply intro-HPC/ch_account.tex]: well, s/should/must/

File: intro-HPC/ch_x2go.tex [hunk 3]
```diff
@@ -68,16 +68,18 @@ \subsection{Option A: direct connection}
...
       not show it by default. The easiest way to access the folder, is by pressing
       \keys{cmd} + \keys{shift} + \keys{g}, which will allow you to enter the name of a directory, which
      you would like to open in Finder. Here, type ``\tilde/.ssh'' and press enter.
      Choose that file and click on \keys{open}.
      \fi
+     \fi
      \ifwindows
-     You should look for your \strong{private} SSH key generated by \keys{puttygen} in \autoref{sec:generate-key-pair}
-     (by default ``\strong{id\_rsa.ppk}'').
+     You should look for your \strong{private} SSH key generated by \keys{puttygen} exportd in ``\emph{OpenSSH}'' format in \autoref{sec:generate-key-pair}
```
Comment (itkovian) [inline intro-HPC/ch_x2go.tex]: `exported`

File: intro-HPC/ch_account.tex [hunk 4]
```diff
@@ -133,13 +133,25 @@ \subsection{How do SSH keys work?}
...
       We recommend using the name ``\strong{id\_rsa.pub}'' for the public key,
       and ``\strong{id\_rsa.ppk}'' for the private key.
+    \item  Finally, save an ``OpenSSH'' version of your private key (in particular for later ``X2Go'' usage)
+      by entering the ``Conversions'' menu and selecting ``Export OpenSSH key''
+      (do \strong{not} select the ``force new file format'' variant).
+      Save the file in the same location as in the previous step
+      with filename ``\strong{id_rsa}''.
+      (If there is no ``Conversions'' menu, you must update your ``puttygen'' version.
+      If you want to do this conversion afterwards, you can start with loading an exsiting ``id_rsa.ppk''
+      and do only this conversions export.)
```
Comment (itkovian) [inline intro-HPC/ch_account.tex]: `only do`

File: intro-HPC/ch_account.tex (hunk 2 above)
Comment (itkovian) [reply intro-HPC/ch_account.tex]: Yes.

**Pull request merged**